src/components/Layout/Auth/EasyLogin.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import kakaotalkLoge from "../../../assets/images/social/kakaotalk-logo.svg";
+import googleLoge from "../../../assets/images/social/google-logo.svg";
+
+interface Image {
+  src: string;
+  alt: string;
+  width: number;
+}
+
+interface Link {
+  href: string;
+  target: string;
+  rel: string;
+  img: Image;
+}
+
+const Links: { [id: string]: Link } = {
+  googleLink: {
+    href: "https://www.google.com/",
+    target: "_blank",
+    rel: "noopener noreferrer",
+    img: {
+      src: googleLoge,
+      alt: "구글",
+      width: 42,
+    },
+  },
+  kakaoLink: {
+    href: "https://www.kakaocorp.com/page/",
+    target: "_blank",
+    rel: "noopener noreferrer",
+    img: {
+      src: kakaotalkLoge,
+      alt: "카카오톡",
+      width: 42,
+    },
+  },
+};
+
+const EasyLogin: React.FC = () => {
+  return (
+    <section className="easy-login">
+      <h3>간편 로그인하기</h3>
+      <div>
+        {Object.entries(Links).map(([key, { href, target, rel, img }]) => (
+          <a key={key} href={href} target={target} rel={rel}>
+            <img src={img.src} alt={img.alt} width={img.width} />
```

Comment:
전달하고자 하는 img 객체의 속성과 <img> 태그의 속성이 일치하므로 아래와 같이 써주셔도 무방합니다.

```suggestion
            <img {...img} />
```

Reply:
와 이렇게도 할 수 있군요!!
그런데 이 방식이 깔끔하고 좋은데 이렇게 해보니까 ESLint 경고가 발생하긴 합니다 ㅠㅠ
"img elements must have an alt prop, either with meaningful text, or an empty string for decorative images."
이렇게 하면 ESLint가 img태그가 alt 속성을 가지고 있는지 알 수 없어서 그런가 보네요😭

Reply:
앗, 아쉽게도 구조분해할당을 사용한 태그가 <img> 여서 약간의 제약이 있네요. 🤔
지금 상황에서는 속성의 갯수가 적어 큰 의미는 없지만 일단 다른 속성들은 넣어주고 alt만 추가적으로 넣어주는 방법도 있는데요. 이 방법으로 ESLint 경고에 대해 통과할 수 있다면 다른 상황에서는 적절하게 사용할 수 있겠습니다. 🤗
```ts
<img {...img} alt={img.alt} />
```